json-schemas/src/account-stats.json

```diff
@@ -27,17 +27,27 @@
         "messages.all.all.count": {
           "type": "number",
           "inclusiveMinimum": 0,
-          "description": "Total message count."
+          "description": "Total number of successful messages, summed over all message types and transports."
         },
         "messages.all.all.data": {
           "type": "number",
           "inclusiveMinimum": 0,
-          "description": "Total message size."
+          "description": "Total message size of all messages successfully sent, summed over all message types and transports."
         },
         "messages.all.all.uncompressedData": {
           "type": "number",
           "inclusiveMinimum": 0,
-          "description": "Total uncompressed message size (excluding any compression, e.g. delta compression: https://ably.com/documentation/realtime/channels/channel-parameters/deltas)."
+          "description": "Total uncompressed message size (excluding any compression, e.g. delta compression: https://ably.com/docs/channels/options/deltas)."
+        },
+        "messages.all.all.failed": {
+          "type": "number",
+          "inclusiveMinimum": 0,
+          "description": "Total number of messages that failed (which did not succeed for some reason other than Ably actively rejecting it, such as rejected by an external integration target, or a service issue on Ably's side)"
+        },
+        "messages.all.all.refused": {
+          "type": "number",
+          "inclusiveMinimum": 0,
+          "description": "Total number of messages that were refused by Ably (eg due to a rate limit, a malformed message, or incorrect permissions on the client's part)"
```

Comment:
"eg" --> "e.g." for consistency throughout. "For example" if we want to match with the style guide.

Reply:
Personally I think the style guide is wrong and we should be using open punctuation in this respect. I'd prefer that we use `eg`, `ie` etc. `e.g.` IMO is a bit last-century.

Reply:
Just to clarify; I meant that in the schema we have a mix of `eg` and `e.g.`, so should opt for one or the other to be consistent (I don't disagree with `eg` being preferable at all, but there were more instances of `e.g.` already).

I'm in the same camp as the style guide though and prefer `for example`.

Reply:
I've replaced with `eg`